.github/workflows/deployDev.yml

```diff
@@ -18,28 +31,47 @@ jobs:
       REACT_APP_VERSION: 'latest'
       REACT_APP_GOOGLE_ANALYTICS_ID: ${{ secrets.GOOGLE_ANALYTICS_ID }}
       REACT_APP_GRAASP_DOMAIN: ${{ secrets.DEV_GRAASP_DOMAIN }}
+    
+    # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
-      - name: Checkout
+      - name: Check out code
         uses: actions/checkout@v2
 
-      - name: Yarn Set Up
+      - name: Setup node
+        id: setup-node
         uses: actions/setup-node@v2
-      - run: yarn install
-      - run: yarn build
+        with:
+          node-version: '14'
+          check-latest: true
+
+      - name: Create .env.dev file
+        run: |
+          touch .env.dev
+          echo REACT_APP_GRAASP_DOMAIN=dev.graasp >> .env.dev
+          echo REACT_APP_GRAASP_APP_ID=${{ secrets.APP_ID }} >> .env.dev
+          echo REACT_APP_MOCK_API=false >> .env.dev
+          cat .env.dev
```

Comment:
It prints right? I don't think it's a good idea!!!

Reply:
Right! it was there for testing, I forgot to remove it